utah-design-system-website/src/react/components/websiteContent/Search.jsx

```diff
@@ -3,6 +3,7 @@ import { useEffect, useState } from 'react';
 
 export function Search() {
   const [query, setQuery] = useState('');
+  const [observer, setObserver] = useState(/** @type {MutationObserver | null} */null);
 
   const insertScript = () => {
     const script = document.createElement('script');
```

Comment:
on line 12, it only inserts the script if the google-search id is not found, but it creates the script element every time. maybe the script element creation should be moved inside the if statement.